src/test/java/com/commercetools/sync/categories/CategorySyncTest.java

```diff
@@ -248,8 +248,8 @@ public void sync_WithExistingCategoryButWithEmptyParentReference_ShouldFailSync(
         assertThat(errorCallBackMessages).hasSize(1);
         assertThat(errorCallBackMessages.get(0)).isEqualTo(format("Failed to resolve references on CategoryDraft with"
             + " key:'key'. Reason: %s: Failed to resolve parent reference on CategoryDraft"
-            + " with key:'key'. Reason: Key is blank (null/empty) on both expanded reference object"
-            + " and reference id field.", ReferenceResolutionException.class.getCanonicalName()));
+            + " with key:'key'. Reason: The value of 'id' field of the Resource Identifier is blank (null/empty).",
```

Comment:
is this needed :(null/empty)?

Reply:
https://github.com/commercetools/commercetools-sync-java/pull/267#discussion_r172524620